doc/conf.py

```diff
@@ -335,17 +335,8 @@ def _check_deps():
 ]
 
 
-class MyWX(MagicMock):
-    class Panel(object):
-        pass
-
-    class ToolBar(object):
-        pass
-
-    class Frame(object):
-        pass
-
-    VERSION_STRING = '2.9'
+class MyCairo(MagicMock):
+    version_info = (1, 2, 0)
```

Comment:
does this need to bump to 1.4.0 now that #10204 is in?

Reply:
yup